Author: Ikuya Awashiro <ikuya@fruitsbasket.info>
Status: applied upstream in r257, can be dropped past 1.4.9
Description: Fix incorrect XIM_QUERY_EXTENSION_REPLY message
https://bugzilla.redhat.com/show_bug.cgi?id=457566
--- a/modules/FrontEnd/IMdkit/i18nIMProto.c
+++ b/modules/FrontEnd/IMdkit/i18nIMProto.c
@@ -106,11 +106,11 @@
 
 static XimFrameRec ext_fr[] =
 {
-    _FRAME(BIT16), 		/* extension major-opcode */
-    _FRAME(BIT16), 		/* extension minor-opcode */
+    _FRAME(BIT8), 		/* extension major-opcode */
+    _FRAME(BIT8), 		/* extension minor-opcode */
     _FRAME(BIT16), 		/* length of extension name */
     _FRAME(BARRAY), 		/* extension name */
-    _PAD4(2),
+    _PAD4(1),
     _FRAME(EOL),
 };
 
